spec/generators/views_generator_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+require "rails_helper"
```

Comment:
When I ran this spec in isolation, it complained about not being able to find a module that Rails provides. I think we missed it before because we were always running this spec alongside others.